### ISSUE TITLE
How to use this repo?

### ISSUE THREAD

**Issue by ohadlevy** [giladger/ElectraWifi#1]
Would it be possible to include more details, e.g:

- [ ] List of required hardware
- [ ] Wiring diagrams and pictures
- [ ] Instruction on how to run this code? (e.g. do you use it with tasmota? mqtt etc?)
- [ ] MQTT example usage (e.g. from Home Assistant)

Thanks!

**Comment by ohadlevy** [comment]
![image](https://user-images.githubusercontent.com/20167/64918628-8ad26580-d7a9-11e9-9e8a-c6ae7794822e.png)
![image](https://user-images.githubusercontent.com/20167/64918634-9aea4500-d7a9-11e9-83b0-5e60db7e5680.png)

**Comment by ohadlevy** [comment]
fixed in 19be57b5ac582c7986c7a57b0c75368577003277

**Comment by guysoft** [comment]
Hey, ia there a picture or documentation how its all connected when complete?  The diagram here and in the main readme shows a ps/2-like connector which I don't seem to see on the unit

**Comment by giladger** [comment]
@guysoft the connector is located at the back of this receiver unit 
![image](https://user-images.githubusercontent.com/40641026/147452862-6205f2a5-97e9-47cd-8caf-dda7f67021db.png)

**Comment by guysoft** [comment]
I can't seem to be able to take it apart. I can see a small plastic bit at the bottom, but it wont pry open when I push it. Nor can I see how its mounted to the wall

**Comment by giladger** [comment]
@guysoft there should be 2 snaps at the top

**Comment by guysoft** [comment]
Indeed,
Ordered the 8-pin mini din connector so I can solder it to the epsp8266 and hope to get it working :)
Here is a photo for reference:
![20211227_143957](https://user-images.githubusercontent.com/325670/147562191-71e25d2d-b6b9-4d2e-9e3b-9667354574b6.jpg)

And the cable.
![20211227_144008](https://user-images.githubusercontent.com/325670/147562189-2f7852e7-cb89-4e6e-80a8-9d486f8ae623.jpg)

**Comment by guysoft** [comment]
Hey,
The schismatic seems to be wrong in two ways:
1. the 8-pin mini din connector data connections need to be reversed
2. The transistor  pins are not labeled. and are location varies between transistors, I used a 2N3904 where the emitter and collector are reversed. 

Here is the schematic that worked for me:
![schematic_fixed](https://user-images.githubusercontent.com/325670/155389599-1939a137-13b6-4464-b4f0-416c43734a36.png)

I think this is the correct one since ariel also confirmed this is the setup that worked for him.

I can PR with the fritzing source and this image if you would like @giladger

**Comment by ohadlevy** [comment]
I will try it out and also report, I wants able to get it working using the prior instructions too.

**Comment by guysoft** [comment]
@ohadlevy After examining the schematic you posted here: https://github.com/giladger/ElectraWifi/issues/1#issuecomment-531544766
The connection to the 8-pin mini din connector is the same as I posted in the schematic I draw. The pin on the AC connector is revered in the one on the git repo currently at: https://github.com/giladger/ElectraWifi/commit/9a471f13f8288fb0cb70ad381657349ce7e8398a#diff-0a47ae9a1ec043feebb0785af3a3722019ba93fe1aaa07b8055781a998a5964b
So it looks like we have something the same :)
You have an extra resistor I don't not sure what for. I have an issue with activating heat on mine, if that might be related.

I can't tell which type of NPN transistor you used, and their pinout is different, I labeled mine so its easier to translate to other parts.
